lantern.go

```diff
@@ -153,12 +157,17 @@ func run(configDir string, user UserConfig) {
 		log.Errorf("Error parsing boolean flag: %v", err)
 	}
 
+	// Need to manually enable logging since we don't have settings in Mobile
+	// TODO: allow configuring whether or not to enable reporting (just like we
+	// already have in desktop)
+	logging.SetReportingEnabled(true)
```

Comment:
Thanks. I guess we'd need to add something to the UI to control this setting, right?